docs/concepts/demisto-sdk-validate.md

```diff
@@ -37,7 +37,7 @@ title: Demisto-SDK Validate checks
 | DO103  | Validate that the given content item's docker-image actually exists in dockerhub       | Platform supported content must have their images available in dockerhub so they can be widely pulled and used.                                                                                                               |               |
 | DO104  | Validate that the given content item has a docker_image.                               | Python and Powershell content run in containers.                                                                                                                                                                              |               |
 | DO105  | Validate that the given content item uses a docker image that is not deprecated        | It is best practice to use images that are maintained by the platform.                                                                                                                                                        |               |
-| DO106  | Validate that the given content-item's docker image isnt outdated'                     | Updated docker images ensure the code dont use outdated dependencies, including bugfixes and fixed vulnerabilities.                                                                                                           | Yes           |
+| DO106  | Validate that the given content-item's docker image isnt outdated'                     | ​Updated Docker images ensure that the code doesn't use outdated dependencies, including bug fixes and fixed vulnerabilities.                                                                                                           | Yes           |
```

Comment:
This document is autogenerated, changes should be done to the source. 
Opened https://github.com/demisto/demisto-sdk/pull/4222, @tschanfeld please review it there.